dash.js

```diff
@@ -179,12 +179,15 @@ var MyDash = new Lang.Class({
     Name: 'DashToDock.MyDash',
 
     _init: function(settings, remoteModel, monitorIndex) {
+        this._dtdSettings = settings;
+
+        // Initialize icon variables and size
         this._maxHeight = -1;
-        this.iconSize = 64;
+        this.iconSize = settings.get_int('dash-max-icon-size');
         this._availableIconSizes = baseIconSizes;
         this._shownInitially = false;
+        this._initializeIconSize(this._dtdSettings.get_int('dash-max-icon-size'));
```

Comment:
and here pass directly this.iconSize...

Reply:
Is this superfluous now?https://github.com/micheleg/dash-to-dock/blob/master/docking.js#L433.

Reply:
Done, both!